web/themes/custom/pineapple/templates/user/user--account--full.html.twig

```diff
@@ -18,28 +18,81 @@
+          {% set cover_image_alt = content.field_u_cover_image|field_target_entity.field_media_image.alt %}
+          <img src="{{ cover_image_url }}" alt="{{ cover_image_alt }}">
         </div>
-        <div class="author-text">
+        <div class="author-name container">
+          <div class="profile-image">
+            {% set profile_image_url = bamboo_render_image_style(content.field_u_profile_image|field_target_entity.field_media_image.entity.uri.value, 'thumbnail_115_x_115') %}
+            {% set profile_image_alt = content.field_u_profile_image|field_target_entity.field_media_image.alt %}
+            <img src="{{ profile_image_url }}" alt="{{ profile_image_alt }}">
+          </div>
           {% include "@components/_heading.twig" with {
             heading_level: 1,
-            heading_attributes: create_attribute().addClass('author-name'),
+            heading_attributes: create_attribute().addClass('subheading'),
             heading_text: content.field_u_name|field_value|first['#context'].value
           } %}
+        </div> <!-- End of author-name-container -->
+        <div class="author-info">
+          {% set user_roles = user.getRoles() %}
+          {% set member_since = user.getCreatedTime()|date("d/m/Y") %}
+          <div class="member-since">
+            <strong>Member since</strong>
+            <p>{{ member_since }}</p>
+          </div>
+          <div class="roles">
+            {% if 'recipe_editor' in user_roles or 'article_editor' in user_roles %}
+              <strong>Role(s)</strong>
+            {% endif %}
+            {% if 'recipe_editor' in user_roles %}
+              <p>Recipe Editor</p>
+            {% endif %}
+            {% if 'article_editor' in user_roles %}
+              <p>Article Editor</p>
+            {% endif %}
+          </div> <!-- End of roles -->
+        </div> <!-- End of author-info -->
+        {% if content.field_u_biography|field_value %}
           <p>{{ content.field_u_biography|field_value }}</p>
-        </div> <!-- End of author-text -->
-      </div> <!-- End of author-info-container -->
-    </div> <!-- End of author-container -->
+        {% endif %}
+        <div class="sidebar-buttons">
+          {% set current_user_id = bamboo_load_currentuser().id %}
+          {% if current_user_id == user.id %}
+            {% set edit_link = '/user/' ~ user.id ~ '/edit'  %}
+            {% include "@components/_link.twig" with {
+              link_text: 'Edit Profile',
+              link_url: edit_link,
+              link_attributes: create_attribute().addClass('btn', 'btn--primary')
+            } %}
+          {% endif %}
+          {% if 'recipe_editor' in user_roles or 'article_editor' in user_roles %}
+            {% include "@components/_link.twig" with {
+              link_text: 'Create Content',
+              link_url: '/node/add',
+              link_attributes: create_attribute().addClass('btn', 'btn--tertiary')
+            } %}
+          {% endif %}
```

Comment:
1. Logged out users should not be able to see this button 
2. People who are not the owner of the profile page should not be able to see this button

Reply:
Logged out users don't have the roles in the if statement.